proto/minder/v1/minder.proto

```diff
@@ -2317,7 +2317,14 @@ message UpdateProjectRequest {
     ];
 
     // description is the description of the project to update.
```

Comment:
This is uninterpreted text, right (not Markdown)?

Reply:
Yes, also confirmed it with Giuseppe 👍